source/Octopus.Tentacle.Tests.Integration/Octopus.Tentacle.Tests.Integration.csproj

```diff
@@ -11,13 +11,17 @@
         <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
         <LangVersion>9</LangVersion>
         <Nullable>annotations</Nullable>
-        <TargetFramework>net6.0</TargetFramework>
+        <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
         <RuntimeIdentifiers>win-x86;win-x64;linux-x64;osx-x64;osx-arm64;linux-arm64;linux-arm;linux-musl-x64</RuntimeIdentifiers>
     </PropertyGroup>
 
     <PropertyGroup Condition=" '$(TargetFramework)' == 'net48' ">
         <DefineConstants>$(DefineConstants);DOES_NOT_SUPPORT_CANCELLATION_ON_SOCKETS</DefineConstants>
     </PropertyGroup>
+    <PropertyGroup Condition="'$(TargetFramework)' == 'net8.0'">
+        <!-- TODO: Investigate why this is needed -->
+        <ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles>
```

Comment:
Was there a failure that led us to add this property?

Reply:
Yes. Remove the property and see. Remove the todo as it only impacts tests

Reply:
I've removed the TODO and replaced with,

<!-- Following the recommended action from https://learn.microsoft.com/en-us/dotnet/core/compatibility/sdk/6.0/duplicate-files-in-output -->